testsuite/tests/typing-modes/lazy.ml

```diff
@@ -109,9 +113,8 @@ let foo (x @ contended) =
 Line 3, characters 6-12:
 3 |     | lazy _ -> ()
           ^^^^^^
-Error: This value is "contended" but is expected to be "uncontended".
-  Hint: In order to force the lazy expression,
-  the lazy needs to be uncontended.
+Error: This value is "contended" but is expected to be "uncontended"
+       because it is a lazy expression that is forced.
```

Comment:
This is not a lazy expression (as in `Texp_lazy`), but rather a value. "it is a forced lazy value" would look nicer.

Reply:
How about "it is a lazy value being forced"? I do want to stress that it's being forced at this location.